ecs-cli/modules/commands/configure/configure_command.go

```diff
@@ -149,24 +166,15 @@ func configureFlags() []cli.Flag {
 			),
 		},
 		cli.StringFlag{
-			Name:  flags.ComposeProjectNamePrefixFlag,
-			Value: flags.ComposeProjectNamePrefixDefaultValue,
-			Usage: fmt.Sprintf(
-				"[Optional] Specifies the prefix added to an ECS task definition created from a compose file. Format <prefix><project-name>.",
-			),
-		},
-		cli.StringFlag{
-			Name:  flags.ComposeServiceNamePrefixFlag,
-			Value: flags.ComposeServiceNamePrefixDefaultValue,
+			Name: flags.ComposeServiceNamePrefixFlag,
 			Usage: fmt.Sprintf(
-				"[Optional] Specifies the prefix added to an ECS service created from a compose file. Format <prefix><project-name>.",
+				"[Deprecated] Specifies the prefix added to an ECS service created from a compose file. Format <prefix><project-name>. (defaults to empty)",
 			),
 		},
 		cli.StringFlag{
-			Name:  flags.CFNStackNamePrefixFlag,
-			Value: flags.CFNStackNamePrefixDefaultValue,
+			Name: flags.CFNStackNameFlag,
 			Usage: fmt.Sprintf(
-				"[Optional] Specifies the prefix added to the AWS CloudFormation stack created on ecs-cli up. Format <prefix><cluster-name>.",
+				"[Optional] Specifies the name of AWS CloudFormation stack created on ecs-cli up. (default: \"amazon-ecs-cli-setup-<cluster-name>\")",
```

Comment:
Specifies the name of the AWS CloudFormation stack created on ecs-cli up. (default: \"amazon-ecs-cli-setup-<cluster-name>\")

Reply:
I'm not sure I understand your suggestion here, the only difference between your string and mine is the removal of <cluster-name>. Since the default includes the customer's cluster name, we need to list that in the help text.

Reply:
Oh, I just realized from my own comment that if you put something in between \<\> Github hides it. I'm guessing that happened in your comment (add \ to escape the special character).

Reply:
OK, thanks for clarifying. Your string looks good, then.